pandas/io/tests/parser/common.py

```diff
@@ -1467,5 +1470,16 @@ def test_memory_map(self):
             'c': ['I', 'II', 'III']
         })
 
-        out = self.read_csv(mmap_file, memory_map=True)
-        tm.assert_frame_equal(out, expected)
+        result = self.read_csv(mmap_file, memory_map=True)
+        tm.assert_frame_equal(result, expected)
+
+    def test_read_csv_utf_aliases(self):
+        # see gh issue 13549
+        expected = pd.DataFrame({'mb_num': [4.8], 'multibyte': ['test']})
+        for byte in [8, 16]:
+            expected.to_csv(path, encoding='utf-' + str(byte), index=False)
```

Comment:
Ah yeah, sorry.  I'm testing the test in a separate file because I haven't figured out how to run the nosetests in common.py yet.